### PR TITLE
{bio}[foss/2022a] GBprocesS v4.0.0.post1, PEAR v0.9.11 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in total)
node3121.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/bf8d1a317027c45987f9f76ad3e2f3c6 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/p/PEAR/PEAR-0.9.11-GCC-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+version = '0.9.11'
+
+homepage = 'https://cme.h-its.org/exelixis/web/software/pear/'
+description = """PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger. 
+ It is fully parallelized and can run with as low as just a few kilobytes of memory."""
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+
+# Needs manual download via web-form,
+# https://www.h-its.org/en/research/sco/software/#NextGenerationSequencingSequenceAnalysis
```
Comment (Micket) [inline easybuild/easyconfigs/p/PEAR/PEAR-0.9.11-GCC-11.3.0.eb:13]:
```suggestion
download_instructions = 'Manual download via web-for '\
    'https://www.h-its.org/en/research/sco/software/#NextGenerationSequencingSequenceAnalysis'
```

**Comment by Micket** [comment]
Thus my suggestions for download instructions :)

File: easybuild/easyconfigs/p/PEAR/PEAR-0.9.11-GCC-11.3.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/p/PEAR/PEAR-0.9.11-GCC-11.3.0.eb]: fixed in 3adbc49977

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
bear-pg0103u11a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6330 CPU @ 2.00GHz (icelake), 1 x NVIDIA NVIDIA A100-PCIE-40GB, 520.61.05, Python 3.6.8
See https://gist.github.com/branfosj/288d1f299bb022a94ffc64348389b685 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**